implementations/rust/ockam/ockam_api/src/control_api/openapi.rs

```diff
@@ -24,12 +26,93 @@ impl Modify for Authentications {
     }
 }
 
+const MAIN_DESCRIPTION: &str = r#"
+## Overview
+This API is designed to control Ockam nodes via HTTP requests without having to use Ockam Command or Ockam library.
```

Comment:
For consistency, let's remove the capital `C` to "Ockam command" (otherwise why not on Ockam Library, Ockam Nodes, etc...)